fv3config/fv3run/_native.py

```diff
@@ -65,6 +66,16 @@ def _set_stacksize_unlimited():
         )
 
 
+def _add_oversubscribe_if_necessary(mpi_flags, n_processes):
+    try:
+        cpu_count = multiprocessing.cpu_count()
+        if cpu_count < n_processes:
+            mpi_flags += ["--oversubscribe"]
+    except NotImplementedError:
+        warnings.warn("could not determine cpu count, assuming it is sufficient")
```

Comment:
replace "it is sufficient" with "number of processors is at least as many as number of MPI tasks"

Reply:
Done, thanks.